pkg/ottl/ottlfuncs/README.md

```diff
@@ -600,6 +601,39 @@ Examples:
 - `Duration("333ms")`
 - `Duration("1000000h")`
 
+
+### ElementizeValuesXML
```

Comment:
Lets do a similar name change for this function.

Reply:
👍 Done